pkg/skaffold/trigger/triggers.go

```diff
@@ -79,10 +84,14 @@ func (t *pollTrigger) Debounce() bool {
 }
 
 func (t *pollTrigger) LogWatchToUser(out io.Writer) {
-	color.Yellow.Fprintf(out, "Watching for changes every %v...\n", t.Interval)
+	if t.isActive() {
+		color.Yellow.Fprintf(out, "Watching for changes every %v...\n", t.Interval)
+	} else {
+		color.Yellow.Fprintln(out, "Not watching for changes...")
+	}
 }
 
-// Start starts a timer.
+// Start starts a timer if active.
```

Comment:
nit: We're still starting the timer with NewTicker but just ignoring it when it goes off, right? Maybe change this comment to be like the other ones you made, saying `// Ignore if trigger is inactive`

Reply:
Done.